coralproject/talk
# Talk Plugin Permalink

# Talk Plugin Permalink

## What's new with this PR?
- Adds `talk-plugin-permalink` as a plugin! :tada:

## How can I test this?
- Pull the latest
- Add `talk-plugin-permalink` to your `plugins.json`
- Check the permalink
- Remove it
- Check that it's not there.
- Check that everything else keeps working as expected.
- Approve this PR
- Thank you, you're awesome.
